<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="bgwidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1261</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget #bgwidget{
background:qlineargradient(spread:pad, x1:0.0298507, y1:0.051, x2:1, y2:0.971364, stop:0 rgba(33, 0, 116, 255), stop:1 rgba(121, 121, 121, 255))
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="welcome">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Login or create a new account</string>
    </property>
   </widget>
   <widget class="QFrame" name="w1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>-20</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#w1{
border-radius:50px;
background:rgba(255, 170, 255,0.2);
}
QWidget#w1:hover{
border-radius:50px;
background:rgba(255, 170, 255,0.7);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="w2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#w2{
border-radius:50px;
background:rgba(170, 170, 255,0.4);
}

QWidget#w2:hover{
border-radius:50px;
background:rgba(170, 170, 255,0.8);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="w3">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>60</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#w3{
border-radius:50px;
background:rgba(175, 197, 255,0.4)
}
QWidget#w3:hover{
border-radius:50px;
background:rgba(175, 197, 255,0.7)
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#login{
background-color:rgba(255, 41, 187,0.5);
color:rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
}
QWidget#login:hover{
background-color:rgba(255, 41, 187,0.9);
color:rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
}
QWidget#login:pressed{
background-color:rgba(255, 255, 255,0.8);
color:rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#register_2{
background-color:rgba(255, 41, 187,0.5);
color:rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
}
QWidget#register_2:hover{
background-color:rgba(255, 41, 187,0.9);
color:rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
}

QWidget#register_2:pressed{
background-color:rgba(255, 255, 255,0.8);
color:rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>signup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goto_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>580</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#goto_2{
background-color:rgba(255, 41, 187,0.5);
color:rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
}
QWidget#goto_2:hover{
background-color:rgba(255, 41, 187,0.9);
color:rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
}

QWidget#goto_2:pressed{
background-color:rgba(255, 255, 255,0.8);
color:rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>No login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
